python/packages/nisar/products/writers/BaseWriterSingleInput.py

```diff
@@ -904,6 +904,55 @@ def copy_from_runconfig(self, h5_field,
 
         return self.set_value(h5_field, data=data, *args, **kwargs)
 
+    def get_value_from_input_runconfig(self, field_name):
```

Comment:
I understand this function to extract the value that corresponds to `field_name`. How about extracting the (dumped) runconfig values from HDF5, parse that using `yaml.save_load()`, and get the field value you want?

Reply:
Thank you, @seongsujeong , I'm afraid a yaml parser will be stricter and start raising exceptions in fields that have nothing to do with `field_name`. Also, we probably would need to do some work to make the H5 dataset `runConfigurationContents` to be parsable by the yaml parser.